main.py

```diff
@@ -0,0 +1,291 @@
+import os
+
+# import sys
+# import yaml
+# import json
+# from slack_bolt.app import App
```

Comment:
Are these all used for the Slack SDK for Python that is commented out lower?

Reply:
Yeah that's correct.